src/types/requestArgs.ts

```diff
@@ -227,10 +227,15 @@ export interface CreateQueryParams extends BaseCRUDParams {
 
 export enum ColumnType {
   Varchar = "varchar",
+  Varbinary = "varbinary",
+  Uint256 = "uint256",
+  Int256 = "int256",
+  Bigint = "bigint",
   Integer = "integer",
   Double = "double",
   Boolean = "boolean",
   Timestamp = "timestamp",
+  Date = "date",
```

Comment:
According to the docs Date is not supported. Do you have reason to believe otherwise? The docs could be outdated. If so, we may want to update those as well.
```suggestion
```

Reply:
I found it in the response of the server when I try to submit an invalid type when creating a table:

```json
{"error":"Failed to deserialize the JSON body into the target type: schema[5].column_type: unknown variant `column_type`, expected one of `varbinary`, `varchar`, `uint256`, `int256`, `integer`, `double`, `boolean`, `timestamp`, `date`, `bigint` at line 1 column 534"}
```

Reply:
Awesome! That works for me! We should probably update the docs repo:

https://github.com/duneanalytics/dune-docs

I can also do this.